software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitSshDriver.java

```diff
@@ -44,6 +45,13 @@
 public class RabbitSshDriver extends AbstractSoftwareProcessSshDriver implements RabbitDriver {
 
     private static final Logger log = LoggerFactory.getLogger(RabbitSshDriver.class);
+
+    // See http://fedoraproject.org/wiki/EPEL/FAQ#howtouse
+    private static final Map<String, String> CENTOS_VERSION_TO_EPEL_VERSION = ImmutableMap.of(
```

Comment:
It would make sense to do this somewhere generic (e.g. `BashCommands`) as EPEL is not specific to Rabbit or even to Erlang. However, I'm ok with merging it here and then it being moved.

I don't like just "waiting for the second usage" before refactoring, because downstream projects may want to use that - those projects should ideally be written without having to change core.